typescript/static-site/static-site.ts

```diff
@@ -59,12 +57,19 @@ export class StaticSite extends cdk.Construct {
         });
         new cdk.CfnOutput(this, 'DistributionId', { value: distribution.distributionId });
 
+        /**
+         * NOTE: the code below is not transpiling properly to JavaScript
```

Comment:
If this gets pulled in, create an issue for it for future work

Reply:
Agreed, leave this in to address later and make an issue referencing this PR and the problem code

Reply:
@NGL321 Ok, I can bring this back but Travis will fail on the build. Is this acceptable?

Reply:
4a200026d143157713619498df8c92e70edd316d

Reply:
It is! Thank you. The ECS construct owners are going to take a look at that

Reply:
Cool. 4a20002 address this.

Reply:
sorry wrong commit.

Reply:
9a9e071cf24d08778302999b6891d412a99163c0